data/en/license_faq.yaml

```diff
@@ -0,0 +1,20 @@
+- Question: Is Cryptomator free to use?
+  Answer: Cryptomator is free to use on Linux, macOS and Windows, but not on iOS and Android. If you purchase a license key for one platform, it is not valid for another. A bundle license does not exist.
+
+- Question: Why is Cryptomator free to use on some platforms, but not on others?
+  Answer: Our licensing strategy differs across platforms because of constraints imposed by Apple App Store policies, Google Play Store policies, and technical differences across desktop operating systems.
+
+- Question: Do I need a supporter certificate to use Cryptomator on desktop?
+  Answer: All functional and encryption features are absolutely free to use on the desktop. By obtaining a supporter certificate, you can unlock the dark mode on desktop apps and also help accelerate Cryptomator's development. Please note, a supporter certificate is not a license key that can be used to unlock mobile apps.
+
+- Question: How can I purchase Cryptomator on Android?
+  Answer: It is a one-time purchase through Google Play or as an APK download from our website or through F-Droid. If you buy the software without going through Google Play, you will receive an Android-only license key. You cannot test the app before purchasing it, but you will be eligible for a refund if desired.
+
+- Question: How can I purchase Cryptomator on Android?
+  Answer: We use a freemium model on iOS. You can download the Cryptomator app from the App Store for free and use it in read-only mode indefinitely. A one-time in-app purchase through the Apple App Store will grant you write access.
+
```

Comment:
There could be a follow-up question for Android:

```
Why is the Android app not freemium?
```

Basically, something like "we're working on it". 😄 See: https://github.com/cryptomator/android/issues/428